components/raftstore/src/engine_store_ffi/mod.rs

```diff
@@ -81,6 +82,30 @@ impl From<&RaftStoreProxy> for RaftStoreProxyPtr {
     }
 }
 
+#[no_mangle]
```

Comment:
```suggestion
#[no_mangle]
```

`#[no_mangle]` can be removed if there is no need to use it outside directly.

Reply:
I think this function will be used outside by TiFlash